README.md

```diff
@@ -40,7 +40,13 @@ $ chmod +x coveralls.phar
 To install php-coveralls with Composer, run the following command:
 
 ```sh
-$ composer require satooshi/php-coveralls --dev
+$ composer require satooshi/php-coveralls
```

Comment:
Why removing the dev flag? Even today we have example of using the development tool at production requirements.